README.md

```diff
@@ -31,14 +31,16 @@ For example: `ci/django.yml` and `ci/properties/django.properties.json`.
 * `categories`: the categories that it will be shown under. Choose at least one category from the list [here](#categories). Further, choose the categories from the list of languages available [here](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml). When a user views the available templates, those templates that match the same language will feature more prominently.
 
 ### Categories
-* continuous-integration
-* deployment
-* testing
+* automation
+* ci
```

Comment:
This can be removed as we have `Continuous-Integration`

Reply:
I was suggesting to make "Categories" consistent with "Directory structure". This repo has directories `automation` and `ci`, so IMO these Categories should remain.

Reply:
directory structure is used while showing the templates. So it is coupled. Category is also used and it not same as directory names

Reply:
I am sorry. In that case I have to reject. Thanks for your contribution